pants

```diff
@@ -412,17 +416,22 @@ function run_bootstrap_tools {
       )
       echo "${parts[*]}"
       ;;
+    bootstrap-version)
+      echo "$SCRIPT_VERSION"
+      ;;
     help|"")
       cat <<EOF
 Usage: PANTS_BOOTSTRAP_TOOLS=1 $0 ...
 
 Subcommands:
   bootstrap-cache-key
     print an opaque that can be used as a key for accurate and safe caching of the pants bootstrap directories
+  bootstrap-version
+    print a version number for the bootstrap script itself
 EOF
       ;;
     *)
-      die "Unknown subcommand for bootstrap tools (due to PANTS_BOOTSTRAP_TOOLS=1): $1 "
+      die "Unknown subcommand for bootstrap tools: $1. Do you mean to run without PANTS_BOOTSTRAP_TOOLS=1? Or, update this script (https://www.pantsbuild.org/docs/installation)?"
```

Comment:
I would like a `bootstrap-update` command, that fetches the latest setup script and saves it over `$0`.

Reply:
That'd be cool! (Although I won't personally work on it, not in the short term, anyway)

Reply:
https://github.com/pantsbuild/setup/issues/131